src/conv/k-g/region_list.h

```diff
@@ -32,24 +32,36 @@
 
 #include "bot.h"
 
+struct Region {
+    Bot                                bot;
+    std::map<std::string, std::string> attributes;
+};
 
 class RegionList {
 public:
     RegionList(void);
 
     Bot& addRegion(const std::string& name);
 
-    Bot& getBot(const std::string& name);
+    Bot&                                getBot(const std::string& name);
+    std::map<std::string, std::string>& getAttributes(const std::string& name) ;
 
     void deleteRegion(const std::string& name);
 
+    void setAttributes(const std::string& name,
+		       const std::map<std::string, std::string>& attributes);
+
+    void writeAttributes(rt_wdb* wdbp,

```

Comment:
I would make this function at least private. But I think, it can be made as a simple static function in region_list.cpp, without mentioning it in the class declaration at all.